flux_led/__init__.py

```diff
@@ -665,6 +671,10 @@ def setRgbw(self, r,g,b,w, persist=True, brightness=None, retry=2):
         msg.append(int(g))
         msg.append(int(b))
         msg.append(int(w))
+
+        if self.protocol == "LEDENET":
```

Comment:
I have default this to upper the user protocol input so by the time it gets here it will always be uppercased.